localstack/services/stepfunctions/asl/component/state/state_execution/state_task/state_task.py

```diff
@@ -86,7 +76,7 @@ def _eval_parameters(self, env: Environment) -> dict:
         # Normalise bindings.
         parameter_normalisers = self._get_parameters_normalising_bindings()
         for parameter_key in list(parameters.keys()):
-            norm_parameter_key = parameter_normalisers.get(parameter_key)
+            norm_parameter_key = parameter_normalisers.get(parameter_key, None)
```

Comment:
was this a stylistic decision? just asking because otherwise this would be redundant :thinking: